### PR TITLE
Updated blogpage .html

### PR DESCRIPTION
**Pull request by ankitmodanwall**
There are few changes that i have specifically considered like 
1. using underline for keeping text highlighted 
2. using Text-align for keep mid positioning for text
3. using break 
4. using div 
5. keep sorted code line for easily understandable code key 
6. using style before text aligning  and most of the line needs to keep highlighted that easily comes to know well.

keep maintaining lines as well.

## Related Issue
[Cite any related issue(s) this pull request addresses. If none, simply state "None”]

## Description
[Please include a brief description of the changes or features added]

## Type of PR

- [x] Bug fix
- [x] Feature enhancement
- [x] Documentation update
- [ ] Other (specify): _______________

## Screenshots / videos (if applicable)
[Attach any relevant screenshots or videos demonstrating the changes]

## Checklist:
- [x] I have performed a self-review of my code
- [x] I have read and followed the Contribution Guidelines.
- [x] I have tested the changes thoroughly before submitting this pull request.
- [x] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [x] I have commented my code, particularly in hard-to-understand areas.

## Additional context:
[Include any additional information or context that might be helpful for reviewers.]

### CODE REVIEW

**Comment by ankitmodanwall** [comment]
Lets find error and keep maintaining lines.

**Comment by Suchitra-Sahoo** [comment]
mention issue number. Add screenshot to demonstrate your changes.

File: blogpage.html [hunk 1]
```diff
@@ -9,21 +9,37 @@
...
+            <span class="material-symbols-outlined" style="font-size">
                 arrow_back
             </span>
         </a>
         
         </div> 
         <div class="navtitle">
-          <h2>Agro<span>Blogs</span></h2>
+          
+          <h2> <u>
```
Comment (ankitmodanwall) [inline blogpage.html:38]: I used to underline all the headlines.

File: blogpage.html [hunk 2]
```diff
@@ -36,17 +52,26 @@ <h2>Agro<span>Blogs</span></h2>
         
          <!--search bar-->
       <div class="search">
-        <input id="input" class="search-bar" placeholder="Search Blogs" oninput="search()"/>
+      
+        <input id="input" class="search-bar" placeholder="Search Blogs" oninput="search()" />
+       <br>
+        <div> <button>click here</button>
```
Comment (ankitmodanwall) [inline blogpage.html:58]: There needed a button bar which helps to finding any result.

File: blogpage.html [hunk 3]
```diff
@@ -9,21 +9,37 @@
...
+    <style>
+  
+  h2{
+ text-align: center;
+ color: black
+  }
+
+
+
+    </style>
```
Comment (ankitmodanwall) [inline blogpage.html:23]: I used to keep center to the all headlines.

**Review by ankitmodanwall — COMMENTED** [review]
I made few changes there too

**Comment by ankitmodanwall** [comment]
You should add up some changes that I have done into your project through
git.

**Comment by Suchitra-Sahoo** [comment]
What is your issue number?

**Pull request closed without merging**